pkg/flowaggregator/flowaggregator.go

```diff
@@ -458,6 +458,10 @@ func (fa *flowAggregator) sendTemplateSet(templateSet ipfixentities.Set, isIPv6
 		ie := ipfixentities.NewInfoElementWithValue(element, nil)
 		elements = append(elements, ie)
 	}
+	fa.set.ResetSet()
+	if err := fa.set.PrepareSet(ipfixentities.Template, templateID); err != nil {
+		return 0, err
+	}
```

Comment:
Why do we need to do reset and prepare here? I saw we had done it [here](https://github.com/antrea-io/antrea/blob/3b0f6e9209874cf8796ea72d5bc5c129e0d63d50/pkg/flowaggregator/flowaggregator.go#L296) and [here](https://github.com/antrea-io/antrea/blob/3b0f6e9209874cf8796ea72d5bc5c129e0d63d50/pkg/flowaggregator/flowaggregator.go#L309-L310)

Reply:
You are right. I encountered the issue when the exporter is restarted in the Flow Aggregator and I see the set still has a record of type `DataType`. Looking more closely at the existing code, we are missing the set reset before L296  [here](https://github.com/antrea-io/antrea/blob/3b0f6e9209874cf8796ea72d5bc5c129e0d63d50/pkg/flowaggregator/flowaggregator.go#L296).

I think it is better to have it in one place. Removed those statements in the caller of `sendTemplateSet`. I am doing the same fix in Flow Exporter too.